gallery/src/components/NxModal/NxModalStackedExample.tsx

```diff
@@ -19,9 +19,9 @@ export default function NxModalStackedExample() {
     <>
       <NxButton onClick={() => setShowModal(true)}>Open modal with stacked example</NxButton>
       {showModal &&
-        <NxModal id="nx-modal-stacked-example" onCancel={modal1CloseHandler}>
+        <NxModal id="nx-modal-stacked-example" onCancel={modal1CloseHandler} aria-labelledby="modal-stacked-header">
```

Comment:
Missing label in the other modal

Reply:
> Missing label in the other modal

Added: https://github.com/sonatype/sonatype-react-shared-components/pull/440/commits/90f65c1b76c9c3e995743f184c788f721810fefa